tests/GitRelease.py

```diff
@@ -51,9 +51,8 @@ def setUp(self):
         with open(self.content_path, "w") as zip_content:
             zip_content.write("Pedro for president.")
 
-        artifact = zipfile.ZipFile(self.artifact_path, "w")
-        artifact.write(self.content_path)
-        artifact.close()
+        with zipfile.ZipFile(self.artifact_path, "w") as artifact:
+            artifact.write(self.content_path)
```

Comment:
This is a nice change, but can you split it out to a separate PR?

Reply:
Can do